decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb

```diff
@@ -1,3 +1,10 @@
+// This file is automatically generated.
+// Do not edit this file, it will be overwritten
```

Comment:
Perhaps I'm missing something, but I don't understand why you say so. AFAIK this file is not automatic, unless you've changed something in the generating process

Reply:
In my development_app, if I change a setting from this file and regenerate it, it gets overridden.  For instance:

1. Replace in the `fontFamily.sans` the "Source Sans Pro" to "Iosevka Term") in `development_app/tailwind.config.js` 
2. `rm -rf development_app/public/decidim-packs/ `
3. `bin/rails assets:precompile` 

If I check the `fontFamily.sans` I see it's "Source Sans Pro".

By following the instructions from this PR (copying the template to `config/tailwind.config.js.erb` and making the change there), then the change is respected

Reply:
Ok, sorry, now I understand it.